css/auth.css

```
@@ -73,11 +66,11 @@
    background-position: center;
    width: 24px;
    height: 24px;
-   top: 34%;
+   top: 42%;
```

💊 제안
이렇게 가로 중앙 정렬하고 싶을때는 아래처럼 하는 것이 더 정확하고 의도를 잘 나타냅니다.

```suggestion
   top: 50%;
   transform: translateY(-50%);
```

다만 이렇게 코드를 변경해주시려면 다른 코드도 수정해주어야합니다! 
지금 해당 태그는 부모.auth-container__password 를 기준으로 position을 잡기 때문에 가로 중앙에 오도록 .auth__input의 maring값을 빼주시고 이를 .auth-container__password로 옮겨주시면 잘 적용이 될 것 같습니다!